app/reducers/channels.js

```diff
@@ -411,158 +494,139 @@ const ACTION_HANDLERS = {
 }
 
 const channelsSelectors = {}
-const channelSelector = state => state.channels.channel
 const channelsSelector = state => state.channels.channels
 const pendingOpenChannelsSelector = state => state.channels.pendingChannels.pending_open_channels
 const pendingClosedChannelsSelector = state =>
   state.channels.pendingChannels.pending_closing_channels
-const pendingWaitingCloseChannelsSelector = state =>
-  state.channels.pendingChannels.waiting_close_channels
 const pendingForceClosedChannelsSelector = state =>
   state.channels.pendingChannels.pending_force_closing_channels
 const waitingCloseChannelsSelector = state => state.channels.pendingChannels.waiting_close_channels
+const closingChannelIdsSelector = state => state.channels.closingChannelIds
 const channelSearchQuerySelector = state => state.channels.searchQuery
 const filterSelector = state => state.channels.filter
 const nodesSelector = state => state.network.nodes
 
-const channelMatchesQuery = (channel, nodes, searchQuery) => {
-  const node = nodes.find(n => channel.remote_pubkey === n.pub_key)
+const channelMatchesQuery = (channelObj, searchQuery) => {
+  if (!searchQuery) {
+    return true
+  }
+
+  const channel = channelObj.channel || channelObj
```

Comment:
I see this pattern a lot. Maybe it's a good idea to get a helper function like `getChannelObject()`

Reply:
Updated